all_repos/push/gitlab_pull_request.py

```diff
@@ -16,6 +16,7 @@ class Settings(NamedTuple):
     api_key: str
     base_url: str = 'https://gitlab.com/api/v4'
     fork: bool = False
+    https: bool = False
```

Comment:
this is unrelated -- and I don't want to enable https cloning -- everything should go over ssh